pkgs/development/interpreters/gnudatalanguage/default.nix

```
@@ -0,0 +1,159 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+# Native build inputs for the compiler
+, cmake
+, enableGAppsWrapping ? true
+, wrapGAppsHook ? null
```

```suggestion
, wrapGAppsHook
```
Why do I want to disable this?